Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -268,16 +268,30 @@ - (NSArray *)accessibilityElements
     return @[];
   }
 
+  if (_accessibilityElements != nil) {
+    return _accessibilityElements;
+  }
+
   if (_accessibilityElements == nil || ASActivateExperimentalFeature(ASExperimentalDisableAccessibilityCache)) {
-    _accessibilityElements = [viewNode accessibilityElements];
+    return [viewNode accessibilityElements];
```

Comment:
This might be the necessary evil (in that we don't cache it and return that next time) if we want to limit the `accessibilityElements` storage to one for either the custom or Texture-defined a11y elements (ie. items returned with current behavior) as opposed to two. It was hard to keep track of whether the ivar contains custom a11y elements or the default especially if you consider the case where you might be dynamically adding/removing subnodes and so you need to update the a11y elements returned for a node. Fwiw, I think another benefit to this method might be the accuracy of the list of items returned from `accessibilityElements`.